### ISSUE TITLE
FLIP pile and FLIP holder yield different results

### ISSUE THREAD

**Issue by 96LawDawg** [ArnoldSmith86/virtualtabletop#1020]
This probably isn't a big deal and it may never matter, but I just wanted to document it.  Add a new holder with a deck of cards so you know the order of them.  Look at the top card.  Add a button to FLIP the holder.  All the cards get flipped but left in the same order.  Use the pile handle flip capability.  The entire pile gets flipped over like you would a physical deck.  The entire pile also gets flipped over if you use a SELECT on the pile as parent and then FLIP the selection.

Bottom line: flipping a pile and flipping a holder do not have the same effect.
